modules/nw-sriov-installing-operator.adoc

```diff
@@ -43,19 +43,14 @@ ifdef::run-level[]
   labels:
     openshift.io/run-level: "1"
 endif::run-level[]
+EOF
```

Comment:
question: where does the `ifdef run-level[]` get defined?

Reply:
@zshi-redhat, at the top of the file, for earlier versions that need this value:

```
ifeval::["{product-version}" == "4.2"]
:run-level:
endif::[]
ifeval::["{product-version}" == "4.3"]
:run-level:
endif::[]
ifeval::["{product-version}" == "4.4"]
:run-level:
endif::[]
```